### PR TITLE
add lebesgue ch4 example

### PR DESCRIPTION
**Pull request by KZHIWEI**

### CODE REVIEW

File: examples/CH4-lebesgue/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Config machine.json file in order to submit task to lebesgue platform.
+
+You can login to lebesgue official websit http://lebesgue.dp.tech/ . Then click [Function]-[DPGEN]-[manual]\(On the top left corner of the function panel\) from left navigator bar http://lebesgue.dp.tech/#/jobs/dpgen.
```
Comment (AnguseZhang) [inline examples/CH4-lebesgue/README.md:3]: Website

File: examples/machine/DeePMD-kit-2.x/lebesgue_v2_machine.json [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+      "machine": {
+        "batch_type": "DpCloudServer",
+        "context_type": "DpCloudServerContext",
+        "local_root" : "./",
+        "remote_profile":{
+          "email": "your lebesgue register email",
+          "password": "your lebesgue password",
+          "program_id": 0,
+            "input_data":{
+                "api_version":2,
```
Comment (AnguseZhang) [inline examples/machine/DeePMD-kit-2.x/lebesgue_v2_machine.json:15]: There is lack of description for `api_version` in `input_data`.One doesn't know the difference between this and `api_version` outside.

**Review by AnguseZhang — CHANGES_REQUESTED** [review]
Thanks for your improvement. I still have some questions in the context.
Besides, one important question is that don't you need to specify`deepmd_version` in `machine.json` if you want to use DP-2.x ? You can search this key in `generator/run.py`.

File: examples/CH4-lebesgue/README.md (hunk 1 above)
Comment (KZHIWEI) [reply examples/CH4-lebesgue/README.md]: fixed

File: examples/machine/DeePMD-kit-2.x/lebesgue_v2_machine.json (hunk 2 above)
Comment (KZHIWEI) [reply examples/machine/DeePMD-kit-2.x/lebesgue_v2_machine.json:16]: thank you for your response, I have added the description about "api_version" in the README file.

**Pull request merged**